### PR TITLE
Feature/card nav

### PR DESCRIPTION
**Pull request by thatbudakguy**
refs #249

something is up with the final test in `test_views.py`; I can see that the correct alt is actually rendered in the context, but the test isn't seeing it. not sure what I'm missing.

### CODE REVIEW

File: mep/people/views.py [hunk 1]
```diff
@@ -523,27 +524,29 @@ def get_breadcrumbs(self):
...
-            prev_card = None
 
+        # get all cards (canvases) from the manifest and store their ids
+        cards = self.object.manifest.canvases
+        card_ids = list(cards.values_list('short_id', flat=True))
+        
+        # create a paginator with 1 card per page and get the current "page"
+        paginator = Paginator(card_ids, 1)
+        current_index = card_ids.index(self.object.short_id)
+        page_obj = paginator.page(current_index + 1) # 1-based page index
```
Comment (rlskoeser) [inline mep/people/views.py]: Can we come up with a better name for `page_obj` ? Maybe something like `card_paginator_page` ? (not great, but  at least more obvious what it is)

File: mep/people/views.py [hunk 2]
```diff
@@ -523,27 +524,29 @@ def get_breadcrumbs(self):
...
+
+        # add next/previous page ids to generate links, if any
+        if page_obj.has_previous():
+            context.update({ 'prev_card_id': card_ids[current_index - 1] })
```
Comment (rlskoeser) [inline mep/people/views.py]: If you're just adding a single value, you don't need to use `update()` — just use `context['prev_card_id'] = ...`

File: mep/people/templates/people/member_card_detail.html [hunk 3]
```diff
@@ -19,26 +19,52 @@
...
         {% else %}
         <span class="next"/></span>
         {% endif %}
     </div>
 
+    <div class="card nav container">
+        <div class="wrapper"> {% spaceless %}
+            <nav class="cards" aria-label="cards">
+                <ol>
+                    {% for card_ in cards.all %}
```
Comment (rlskoeser) [inline mep/people/templates/people/member_card_detail.html]: Can we find a better variable name than `card_` here? Kind of hard to distinguish `card_` and `card`.  Thumbnail? Canvas?

File: mep/people/views.py (hunk 1 above)
Comment (thatbudakguy) [reply mep/people/views.py]: does `card_page` make sense?

File: mep/people/templates/people/member_card_detail.html (hunk 3 above)
Comment (thatbudakguy) [reply mep/people/templates/people/member_card_detail.html]: I struggled with this, but I like Canvas!

File: mep/people/views.py (hunk 1 above)
Comment (rlskoeser) [reply mep/people/views.py]: ... I think that sounds ok. Better than `page_obj` !

**Pull request merged**